src/components/ContentSidebar/__tests__/__snapshots__/Sidebar-test.js.snap

```diff
@@ -32,6 +32,6 @@ exports[`components/ContentSidebar/Skills/Sidebar should render skills sidebar 1
     hasSkills={true}
     selectedView="skills"
   />
-  <SkillsSidebar />
+  <Component />
```

Comment:
Any idea why jest does this occasionally? I find I need to stub the component manually for the name to actually show up